.github/workflows/system-tests.yml

```diff
@@ -116,4 +114,10 @@ jobs:
           WALLET_BITCOIN_WIF: ${{ secrets.SYSTEM_TESTS_WALLET_PRIVATE_KEY_WIF }}
           DEPOSITOR_BITCOIN_WIF: ${{ secrets.SYSTEM_TESTS_DEPOSITOR_PRIVATE_KEY_WIF }}
         run: |
+          # As system tests use the same Bitcoin key and perform the same initial
+          # actions, they often retrieve UTXOs simultaneously. This can lead to
+          # issues where both tests try to spend the same UTXOs, causing
+          # conflicts. Adding a delay helps to mitigate this problem by allowing
+          # time for the UTXO list to update.
+          sleep 240
```

Comment:
Maybe an idea is making those tests sequential? We can do it by adding `needs: system-tests-deposit-redemption` on top of the `system-tests-minting-unminting` job definition. I know the execution time will be affected but, if those tests affect each other, this may be a good move.

Reply:
Done in 0c183c3c580fd584945d79c258dd74a610467ec3.